frontend/GameGUI.py

```diff
@@ -23,12 +24,13 @@ def check_events(self):
                 sys.exit()
 
     def run_game(self):
+        self.board.build_board() #does this need to be called constantly?
```

Comment:
Its only called once right? At the beginning of the program?

Reply:
Its outside the while loop

Reply:
It is now -- that was a comment I forgot to remove when I added the pull request, since that line was initially in the while loop. I removed it from my local branch immediately after, but that doesn't reflect here. My bad!

Reply:
I'll remove it when I push the image/example README change.